ddp-workspace/projects/ddp-testboston/src/app/components/welcome/welcome.component.scss

```diff
@@ -100,6 +100,10 @@
     border-top: 1px solid mat-color($app-theme, 500);
 }
 
+.mat-expansion-panel-header {
+    transition: 0.3s;
+}
```

Comment:
This change does not apply to the modal window. Just added a smooth hover animation for the expansion panel on the home page.